nfs.go

```diff
@@ -49,14 +59,40 @@ var zeroHandle handle
 var statusHandle = handle{0: 'S', 1: 'T', 2: 'A', 3: 'T', 4: 'U', 5: 'S'}
 
 type NFSHandler struct {
-	fs *FS
-
+	fs          *FS
 	nfs.Handler // temporary embedding during dev to watch what panics
 
-	mu          sync.Mutex
-	handle      map[handle]handleTarget
-	readCache   map[handle]*readCacheEntry
+	metricFileContentCacheHit  expvar.Int
```

Comment:
These can't be viewed at the moment, right?

Reply:
they can now!